airflow/cli/commands/task_command.py

```diff
@@ -69,6 +70,18 @@
 CreateIfNecessary = Union[Literal[False], Literal["db"], Literal["memory"]]
 
 
+def _set_task_deferred_context_var():
+    """
+    Tell task log handler that task exited with deferral.
+
+    :meta private:
+    """
+    logger = logging.getLogger()
+    h = next((h for h in logger.handlers if hasattr(h, "ctx_task_deferred")), None)
+    if h:
+        h.ctx_task_deferred = True
```

Comment:
curious, is the rationale here to avoid the "truthy" evaluation on h?

Reply:
or just readability?

Reply:
Yeah the main thing is truthy evaluation, or any evaluation really. This is probably not an issue for `logger.handlers` in practice, but it’s better to avoid readers needing to consider the posibility entirely.